### PR TITLE
Docs: Fix document possible value missed of 45 resources

### PR DESCRIPTION
**Pull request by wuxu92**
This PR is to fix the documents in which the property `ValidateFunc` is `StringInSlice` but some values are missed in the document.

### CODE REVIEW

**Review by stephybun — CHANGES_REQUESTED** [review]
Thanks for this @wuxu92, I left a review comment over in #19271 which is relevant for this PR as well, would you mind applying those suggestions to this one as well? Thanks!

**Comment by wuxu92** [comment]
This PR only adds the missed values, it's different from 19271. I have updated both of them though.

File: website/docs/r/iot_time_series_insights_standard_environment.html.markdown [hunk 1]
```diff
@@ -36,7 +36,7 @@ The following arguments are supported:
 
 * `location` - (Required) Specifies the supported Azure location where the resource exists. Changing this forces a new resource to be created.
 
-* `sku_name` - (Required) Specifies the SKU Name for this IoT Time Series Insights Standard Environment. It is string consisting of two parts separated by an underscore(\_).The fist part is the `name`, valid values include: `S1` and `S2`. The second part is the `capacity` (e.g. the number of deployed units of the `sku`), which must be a positive `integer` (e.g. `S1_1`). Changing this forces a new resource to be created.
+* `sku_name` - (Required) Specifies the SKU Name for this IoT Time Series Insights Standard Environment. It is string consisting of two parts separated by an underscore(\_).The fist part is the `name`, valid values include: `S1_1`, `S1_2`, `S1_3`, `S1_4`, `S1_5`, `S1_6`, `S1_7`, `S1_8`, `S1_9`, `S1_10`, `S2_1`, `S2_2`, `S2_3`, `S2_4`, `S2_5`, `S2_6`, `S2_7`, `S2_8`, `S2_9` and `S2_10`. The second part is the `capacity` (e.g. the number of deployed units of the `sku`), which must be a positive `integer` (e.g. `S1_1`). Changing this forces a new resource to be created.
```
Comment (stephybun) [inline website/docs/r/iot_time_series_insights_standard_environment.html.markdown:39]:
```suggestion
* `sku_name` - (Required) Specifies the SKU Name for this IoT Time Series Insights Standard Environment. It is string consisting of two parts separated by an underscore(\_).The first part is the `name`, valid values include: `S1_1`, `S1_2`, `S1_3`, `S1_4`, `S1_5`, `S1_6`, `S1_7`, `S1_8`, `S1_9`, `S1_10`, `S2_1`, `S2_2`, `S2_3`, `S2_4`, `S2_5`, `S2_6`, `S2_7`, `S2_8`, `S2_9` and `S2_10`. The second part is the `capacity` (e.g. the number of deployed units of the `sku`), which must be a positive `integer` (e.g. `S1_1`). Changing this forces a new resource to be created.
```

File: website/docs/r/mssql_managed_instance.html.markdown [hunk 2]
```diff
@@ -211,7 +211,7 @@ The following arguments are supported:
 
 * `administrator_login_password` - (Required) The password associated with the `administrator_login` user. Needs to comply with Azure's [Password Policy](https://msdn.microsoft.com/library/ms161959.aspx)
 
-* `license_type` - (Required) What type of license the Managed Instance will use. Valid values include can be `PriceIncluded` or `BasePrice`.
+* `license_type` - (Required) What type of license the Managed Instance will use. Valid values include can be `LicenseIncluded` and `BasePrice`.
```
Comment (stephybun) [inline website/docs/r/mssql_managed_instance.html.markdown:214]:
```suggestion
* `license_type` - (Required) What type of license the Managed Instance will use. Possible values are `LicenseIncluded` and `BasePrice`.
```

File: website/docs/r/mssql_server_security_alert_policy.html.markdown [hunk 3]
```diff
@@ -60,7 +60,7 @@ The following arguments are supported:
 
 * `server_name` - (Required) Specifies the name of the MS SQL Server. Changing this forces a new resource to be created.
 
-* `state` - (Required) Specifies the state of the policy, whether it is enabled or disabled or a policy has not been applied yet on the specific database server. Allowed values are: `Disabled`, `Enabled`.
+* `state` - (Required) Specifies the state of the policy, whether it is enabled or disabled or a policy has not been applied yet on the specific database server. Allowed values are: `Disabled`, `Enabled` and `New`.
```
Comment (stephybun) [inline website/docs/r/mssql_server_security_alert_policy.html.markdown:63]:
```suggestion
* `state` - (Required) Specifies the state of the policy, whether it is enabled or disabled or a policy has not been applied yet on the specific database server. Possible values are `Disabled`, `Enabled` and `New`.
```

File: website/docs/r/synapse_spark_pool.html.markdown [hunk 4]
```diff
@@ -91,9 +91,9 @@ The following arguments are supported:
 
 * `synapse_workspace_id` - (Required) The ID of the Synapse Workspace where the Synapse Spark Pool should exist. Changing this forces a new Synapse Spark Pool to be created.
 
-* `node_size_family` - (Required) The kind of nodes that the Spark Pool provides. Possible value is `MemoryOptimized`.
+* `node_size_family` - (Required) The kind of nodes that the Spark Pool provides. Possible value is `MemoryOptimized` and `None`.
```
Comment (stephybun) [inline website/docs/r/synapse_spark_pool.html.markdown:94]:
```suggestion
* `node_size_family` - (Required) The kind of nodes that the Spark Pool provides. Possible values are `MemoryOptimized` and `None`.
```

File: website/docs/r/synapse_spark_pool.html.markdown [hunk 5]
```diff
@@ -91,9 +91,9 @@ The following arguments are supported:
...
 
-* `node_size` - (Required) The level of node in the Spark Pool. Possible value is `Small`, `Medium` and `Large`.
+* `node_size` - (Required) The level of node in the Spark Pool. Possible value is `Small`, `Medium`, `Large`, `None`, `XLarge`, `XXLarge` and `XXXLarge`.
```
Comment (stephybun) [inline website/docs/r/synapse_spark_pool.html.markdown:96]:
```suggestion
* `node_size` - (Required) The level of node in the Spark Pool. Possible values are `Small`, `Medium`, `Large`, `None`, `XLarge`, `XXLarge` and `XXXLarge`.
```

File: website/docs/r/synapse_sql_pool_security_alert_policy.html.markdown [hunk 6]
```diff
@@ -91,7 +91,7 @@ The following arguments are supported:
 
 * `sql_pool_id` - (Required) Specifies the ID of the Synapse SQL Pool. Changing this forces a new resource to be created.
 
-* `policy_state` - (Required) Specifies the state of the policy, whether it is enabled or disabled or a policy has not been applied yet on the specific SQL pool. Allowed values are: `Disabled`, `Enabled`.
+* `policy_state` - (Required) Specifies the state of the policy, whether it is enabled or disabled or a policy has not been applied yet on the specific SQL pool. Allowed values are: `Disabled`, `Enabled` and `New`.
```
Comment (stephybun) [inline website/docs/r/synapse_sql_pool_security_alert_policy.html.markdown:94]:
```suggestion
* `policy_state` - (Required) Specifies the state of the policy, whether it is enabled or disabled or a policy has not been applied yet on the specific SQL pool. Possible values are `Disabled`, `Enabled` and `New`.
```

File: website/docs/r/synapse_workspace_security_alert_policy.html.markdown [hunk 7]
```diff
@@ -84,7 +84,7 @@ The following arguments are supported:
 
 * `synapse_workspace_id` - (Required) Specifies the ID of the Synapse Workspace. Changing this forces a new resource to be created.
 
-* `policy_state` - (Required) Specifies the state of the policy, whether it is enabled or disabled or a policy has not been applied yet on the specific workspace. Allowed values are: `Disabled`, `Enabled`.
+* `policy_state` - (Required) Specifies the state of the policy, whether it is enabled or disabled or a policy has not been applied yet on the specific workspace. Allowed values are: `Disabled`, `Enabled` and `New`.
```
Comment (stephybun) [inline website/docs/r/synapse_workspace_security_alert_policy.html.markdown:87]:
```suggestion
* `policy_state` - (Required) Specifies the state of the policy, whether it is enabled or disabled or a policy has not been applied yet on the specific workspace. Possible values are `Disabled`, `Enabled` and `New`.
```

File: website/docs/r/windows_function_app_slot.html.markdown [hunk 8]
```diff
@@ -387,7 +387,7 @@ An `application_stack` block supports the following:
 
 * `java_version` - (Optional) The version of Java to use. Possible values are `8`, `11` & `17` (In-Preview).
 
-* `node_version` - (Optional) The version of Node to use. Possible values include `12`, and `14`
+* `node_version` - (Optional) The version of Node to use. Possible values include `~12`, `~14`, `~16` and `~18`.
```
Comment (stephybun) [inline website/docs/r/windows_function_app_slot.html.markdown:390]:
```suggestion
* `node_version` - (Optional) The version of Node to use. Possible values are `~12`, `~14`, `~16` and `~18`.
```

**Review by stephybun — APPROVED** [review]
Thanks @wuxu92 LGTM 🌻

**Pull request merged**